docs/tutorials/using-ev3-speaker.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using the EV3 Speaker
+group: hardware-standard
+---
+
+* Table of Contents
+{:toc}
+
+
+## Introduction
+
+The EV3 uses pulse width modulation (PWM) to drive an amplifier that produces the
```

Comment:
I disagree. "pules width modulation" is not a proper noun, so it does not make sense to capitalize it here.